kpops/cli/registry.py

```diff
@@ -53,11 +55,18 @@ def find_class(module_name: str, baseclass: type[T]) -> type[T]:
 
 def _find_classes(module_name: str, baseclass: type[T]) -> Iterator[type[T]]:
     module = importlib.import_module(module_name)
+    if module.__file__ and not module_name.startswith(KPOPS_MODULE):
+        file_path = Path(module.__file__)
+        log.debug(f"Picked up: {file_path.resolve().relative_to(Path.cwd())}")
```

Comment:
tbh, I'd prefer printing the filepath as-is, without resolving. If there a symbolic links and the like, as a user I would wanna see the link, not the resolved path

Reply:
tbh, I prefer to have it like that. Basically we are showing the files paths from the `pipeline_base_dir` instead of the root. The parent path of the `pipeline_base_dir` is less useful for the user. With all the changes, the logs would look like this:

```console
Yaml - Picked up: config.yaml
Registry - Picked up: tests/pipeline/test_components/__init__.py
Yaml - Picked up: tests/pipeline/resources/pipeline-with-env-defaults/pipeline.yaml
Yaml - Picked up: tests/pipeline/resources/pipeline-with-env-defaults/defaults_development.yaml
Yaml - Picked up: tests/pipeline/resources/pipeline-with-env-defaults/defaults.yaml
BaseDefaultsComponent - Found defaults for component type streams-app in defaults.yaml
Yaml - Picked up: tests/pipeline/resources/defaults.yaml
BaseDefaultsComponent - Found defaults for component type streams-app in defaults.yaml
Yaml - Picked up: tests/pipeline/defaults.yaml
BaseDefaultsComponent - Found defaults for component type kafka-app in defaults.yaml
BaseDefaultsComponent - Found defaults for component type kafka-app in defaults.yaml
BaseDefaultsComponent - Found defaults for component type kubernetes-app in defaults_development.yaml
BaseDefaultsComponent - Found defaults for component type kubernetes-app in defaults.yaml
BaseDefaultsComponent - Found defaults for component type kubernetes-app in defaults.yaml
BaseDefaultsComponent - Found defaults for component type kubernetes-app in defaults.yaml
BaseDefaultsComponent - Found defaults for component type pipeline-component in defaults.yaml
BaseDefaultsComponent - Enriching component of type streams-app
```

Reply:
but why do we have to resolve anything at all? isn't the path already relative an what we want?

Reply:
Good point! in the registry we don't really need to resolve because we are getting the `Path(module.__file)`.